templates/scenarios/js/command-and-response/README.md

```diff
@@ -28,7 +28,7 @@ The app template is built using the TeamsFx SDK, which provides a simple set of
 >
 > To run the command bot template in your local dev machine, you will need:
 >
-> - `Node.js` installed locally (recommended version: 16)
+> - [Node.js](https://nodejs.org/en/), supported versions: 14, 16, 18
```

Comment:
`https://nodejs.org/` ? Just remove the locale part.

Reply:
Good point! Now all the readme files and documents use https://nodejs.org/en/. I will change all of them in aother PR.